APR-Core/Core.lua

```diff
@@ -77,20 +77,54 @@ function APR.AutoPathOnBeta(ChoiceZ) -- For the Speed run button
 			APR.RoutePlan.FG1["Fxz2Custom"..CLi]:Hide()
 		end
 	end
-	if (ChoiceZ == 1 and (ZeMap == 1409 or ZeMap == 1726 or ZeMap == 1727 or ZeMap == 1728) and APR.Faction == "Alliance") then
+	if (routeChoice == 2 and APR.Level < 50 and APR.Faction == "Alliance") then
 		autoPatchStartInsertRoute()
-		tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"01-10 Exile's Reach")
-		tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"(1/8) 10-50 Stormwind")
-		tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"(2/8) 10-50 Tanaan Jungle")
-		tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"(3/8) 10-50 Shadowmoon")
-		tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"(4/8) 10-50 Gorgrond")
-		tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"(5/8) 10-50 Talador")
-		tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"(6/8) 10-50 Shadowmoon")
-		tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"(7/8) 10-50 Talador")
-		tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"(8/8) 10-50 Spires of Arak")
-		tinsert(APR_Custom[APR.Name.."-"..APR.Realm],"DEV - AStoryMode Only (Not Enough XP)")
-	elseif (ChoiceZ == 1 and APR.Level < 60 and APR.Level > 9 and APR.Faction == "Alliance") then
+		if(APR.Level < 10 and (ZeMap == 1409 or ZeMap == 1726 or ZeMap == 1727 or ZeMap == 1728)) then

```

Comment:
UIMapID 1728 is The Runecarver in Torghast
It doesn't belong here. Not that it will ever check, but might as well remove it now.